fdbclient/NativeAPI.actor.h

```diff
@@ -407,5 +407,10 @@ ACTOR Future<bool> checkSafeExclusions(Database cx, vector<AddressExclusion> exc
 inline uint64_t getWriteOperationCost(uint64_t bytes) {
 	return bytes / std::max(1, CLIENT_KNOBS->WRITE_COST_BYTE_FACTOR) + 1;
 }
+
+// The quiet database check (which runs at the end of every test) will always time out due to active data movement.
```

Comment:
Some tests also run the quiet database check at the start, if waitForQuiescenceBegin is enabled.
In that case, you'd disable the wiggle for the entire test, which doesn't seem desired.

And once you fix that, to ensure something else isn't preventing the wiggle from being properly tested, I'd also suggest adding some test coverage statements for important parts of the perpetual wiggle (excluding/including/pause, etc...) so you can verify that your simulation tests are actually covering all of the behavior you want to test.

Reply:
I agree. But I think in QuietDatabase actor we still need to disable wiggling without enabling it again. Otherwise, wait(QuietDatabase) doesn't really return a quiet database. I'm trying to figure out a way to properly restore the setting of perpetual wiggling in some test cases, but might in another PR. Can we leave it like now for this PR?

Add coverage is a good idea, will add in this PR.

Reply:
Yeah that makes sense. It seems fine to me to do that in a follow-up PR, but it might be good to check the coverage before you merge to make sure it's getting tested sufficiently in simulation as is.